aristanetworks/ansible-avd
# Fix(eos_designs): Fix and test custom python modules for ip addressing and interface descriptions

## Change Summary


Fix and test custom python modules for ip addressing and interface descriptions

## Component(s) name

`arista.avd.eos_designs`

## Proposed changes



- Add molecule coverage for custom python modules for ip_addressing and interface_descriptions.
- Fix issue in eos_designs_facts to load the configured modules correctly

## How to test



Added in molecule.
First commit use existing Jinja templates set in custom_templates.
Next refactor to custom python modules and check that nothing changes.

## Checklist

### User Checklist


- N/A

### Repository Checklist
